miniapps/nurbs/nurbs_ex24.cpp

```diff
@@ -126,8 +126,8 @@ int main(int argc, char *argv[])
       }
    }
 
-   // 5. Define a finite element space on the mesh. Here we use Nedelec or
-   //    Raviart-Thomas finite elements of the specified order.
+   // 5. Define a finite element space on the mesh. Here we use H_1, H(Div) or
```

Comment:
For consistency, `H^1` is used above on line 26.

Reply:
Fixed